azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/redux/action/CallDiagnosticsAction.kt

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.calling.redux.action
+
+import com.azure.android.communication.ui.calling.models.MediaCallDiagnosticModel
+import com.azure.android.communication.ui.calling.models.NetworkCallDiagnosticModel
+import com.azure.android.communication.ui.calling.models.NetworkQualityCallDiagnosticModel
+
+internal sealed class CallDiagnosticsAction : Action {
+    class NetworkQualityCallDiagnosticsUpdated(val networkQualityUpdated: NetworkQualityCallDiagnosticModel) : CallDiagnosticsAction()
```

Comment:
Why property is named after class NetworkQualityCallDiagnosticsUpdated?
It should be after NetworkQualityCallDiagnosticModel: networkQualityDiagnosticsModel

Same for all other actions

Reply:
Fixed the naming for all the action parameters